lib/features/down_by_maintenance/data/services/mock_check_server_status_service.dart

```diff
@@ -0,0 +1,36 @@
+import 'dart:async';
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter_template/features/down_by_maintenance/domain/entities/check_result.dart';
+import 'package:flutter_template/features/down_by_maintenance/domain/services/check_server_status_service.dart';
+
+/// Implementation of [ICheckServerStatusService] that mocks server status.
+class MockCheckServerStatusService implements ICheckServerStatusService {
+  @override
+  final ValueNotifier<ServerCheckResult> serverStatus = ValueNotifier(ServerCheckResult.processing);
```

Comment:
It is better to replace it with `BehaviorSubject`.

Reply:
\+